src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenByIntegratedWindowsAuthParameterBuilder.cs

```diff
@@ -46,6 +46,18 @@ public AcquireTokenByIntegratedWindowsAuthParameterBuilder WithUsername(string u
             return this;
         }
 
+        /// <summary>
+        /// Enables MSAL to read the federation metada for a wstrust exchange from a file instead of acquiring it from an endpoint
+        /// See aka.ms/MsalFederationMetadata
```

Comment:
https://aka.ms/... Also I don't think you created it.

Reply:
working on documentation for this